challenges.yml

```
@@ -2088,3 +2088,217 @@ extending_prop_transition:
+    > acts as a parameter of the `v-model` *"function"* and tells it that
+    > the prop we want to use is called `value`. Just like in Vue 2!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @input="$emit('input', $event.target.value)"
+          >
+          ```
+        b: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @input="$emit('input', $event)"
+          >
+          ```
+        c: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @change="$emit('input', $event.target.value)"
+          >
+          ```
+        d: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @keyup="$emit('input', $event.target.value)"
+          >
+          ```
+      correct: a
+extend_v_bind_object:
+  question: |
+    `v-bind` is an amazing shortcut that lets us assign multiple, or even dynamic
+    props to a component using one single attribute. Given this piece of code,
+    what are the actual props that the child component will receive?
+
+    ```vue
+    <template>
+        <child-component
+            text="Some text prop here"
+            v-bind="{
+                isValid: true,
+                showText: true,
+            }"
+        />
+    </template>
+    ```
+  explanation: |
+    The fact that `v-bind` is so flexible, is a great feature of Vue. You can
+    mix `v-bind` with regular props and get both passed down to the child
+    component.
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```
+          {
+              isValid: true,
+              showText: true,
+          }
+          ```
+        b: |
+          ```
+          {
+              text: 'Some text prop here',
+          }
+          ```
+        c: |
+          ```
+          {
+              isValid: true,
+              showText: true,
+              text: 'Some text prop here',
+          }
+          ```
+      correct: c
+
+extending_checkout_submit:
+  question: |
+    In this video, we show a neat trick that we can do when submitting a form
+    in Vue:
+
+    ```vue
+    <form @submit.prevent="onSubmit">
+        .
+        .
+        .
+    </form>
+    ```
+
+    What exactly does the modifier `prevent` do?
+  explanation: |
+    Event listeners in Vue have several modifiers, of which `prevent` is
+    the most useful!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          It prevents our component from freezing if there are any errors
```

```suggestion
          It prevents our component from breaking if there are any errors
```

This just seemed like a more difficult, tricky way to answer this :)